models.js

```diff
@@ -17,4 +17,12 @@ exports.selectArticleById = (articleId) => database(`
 SELECT * FROM articles WHERE article_id = $1;`, [articleId])
     .then(({ rows: articles }) => articles.length
         ? articles
-        : Promise.reject({ status: 404, message: 'Article Not Found' }))
\ No newline at end of file
+        : Promise.reject({ reason: 'NoArticle' }))
+
+exports.selectCommentsByArticle = (articleId) => database(`
+SELECT comment_id, votes, created_at, author, body FROM comments 
+WHERE article_id = $1 
+ORDER BY created_at DESC;`, [articleId])
+    .then(({ rows: comments }) => comments.length
+        ? comments
+        : Promise.reject({ reason: 'NoArticle' }))
```

Comment:
Generally within our promise.reject we want to include two keys - 'status' and 'msg' so that both the code and the msg can be created in our models and our custom error handler can access these values which could change situationally. Have a look at your error handler and make sure it is intaking a dynamic status code and msg and look at refactoring this please :)

Reply:
Thank you for the feedback. I've updated it now.